### DISCUSSION TITLE
`harlequin --help` has a poor contrast on light-themed terminal

### DISCUSSION THREAD

**Discussion by glujan** [tconbeer/harlequin#549] [Q&A, Troubleshooting, Support]
While I enjoy the aesthetic of the harlequin website, it unfortunately does not play nicely with a light-themed terminal. I'm using a default terminal with light mode in Ubuntu 22.04 and running `harlequin --help` gives such an output:
![Output of --help on light terminal theme](https://github.com/tconbeer/harlequin/assets/3079032/b22b4a8e-eae2-434d-847c-177e7dd2bcc8)

As you can see, the yellow text is barely visible.

Depending on maintainer priorities, this may be more of a feature request or a bug report, I've decided to share it as a feature idea. Maybe harlequin could _guess_ if it is run in dark or light mode, and apply a more suitable theme, or a current color theme could be tweaked.

**Comment by tconbeer** [comment]
This applies to `harlequin --config` as well. (although unfortunately it'll have to be solved for differently)

It's hard to detect terminal background colors/themes, but there are maybe some things we can do: https://github.com/Textualize/rich/discussions/1170

Thankfully `--help` does support `NO_COLOR`, so this is a temp workaround:

```bash
$ NO_COLOR=1 harlequin --help
```

Sadly the NO_COLOR workaround does not work for `--config`.

**Reply by tconbeer** [reply]
Opened #552 to create a workaround for `--config`
